pkg/telemetry/marketing/telemeter.go

```diff
@@ -13,6 +13,7 @@ type Telemeter interface {
 type Config struct {
 	ID       string
 	OrgID    string
+	SKU      string
```

Comment:
I used the SKU as example of something could be interesting to correlate. We don't need to send the SKU in Central itself, but I think it would be worthwhile to send the tenant id as a correlation id in Central. That way, we can correlate data from fleet-manager or the console UI (like the SKU, or something else).

Reply:
What would we `SKU` work for self-hosted installs of ACS ?

Reply:
We don't really have access to the SKU from self-hosted Centrals. In any case we shouldn't capture the SKU from within Central, I just used it as an example of something that we might want to correlate to using the Central tenant ID.

Reply:
Let me give another example of why I think tenant correlation is useful: At some point, someone (maybe us, maybe someone else) will provide business metrics for the ACS cloud service - for example, how much money we are making and how we are spending in infrastructure per tenant. It would be nice to be able to correlate the user behavior of a tenant with the revenue/profit it generates us. This way we would be able to improve our product based on what makes the most financial sense.

Reply:
Please no unrelated junk. Centrals have no concept of SKU and vaguely of org ID. If you want to forward arbitrary data from, say, Central deployment, to telemetry store, please use a map here so things like `SKU`s are contained in a single place like `GetDeviceConfig()`.